app/views/admin/neighbourhoods/_form.html.erb

```diff
@@ -34,44 +33,13 @@
   <h3>Ward</h3>
   <div class="row">
     <div class="col-md-6">
-      <%= f.input :WD19CD, label: 'WD19CD', readonly: true, as: :vue_string %>
+      <%= f.input :unit_code_value, label: 'WD19CD', readonly: true %>
     </div>
     <div class="col-md-6">
-      <%= f.input :WD19NM, label: 'WD19NM', readonly: true, as: :vue_string %>
```

Comment:
"vue_string" is unable to properly access this value for whatever reason, so it got removed from the property.

Reply:
that is weird and could be an indication something is up with javascript builds?

Reply:
hmm. I know that I didn't convert the Javascript picker, but specifically because that's defined in another task. Might be worth making a small ticket for it (?)